Jenkinsfile

```diff
@@ -17,6 +17,13 @@
  * License-Filename: LICENSE
  */
 
+/**
+ * Required Jenkins plugins:
+ * - https://plugins.jenkins.io/pipeline-utility-steps/
+ */
```

Comment:
Out of curiosity, for which part of the new code is the plugin required? Should this be documented in the README?

Reply:
> Out of curiosity, for which part of the new code is the plugin required?

It's required for the `readJSON` function. Which reminds me that I should probably use [readYaml](https://github.com/jenkinsci/pipeline-utility-steps-plugin/blob/master/docs/STEPS.md#configuration-files) instead, as we usually prefer YAML, and it tends to have the smaller file size.

> Should this be documented in the README?

I'd rather document / maintain this in a single place, and that should be the `Jenkinsfile` so you can easily see the information also from within Jenkins. But I'll add a general comment to the `README.md`.